Documentation/runtime-configuration.md

```diff
@@ -161,3 +161,9 @@ $ etcd
 etcd: this member has been permanently removed from the cluster. Exiting.
 exit 1
 ```
+
+#### Strict Reconfiguration Check Mode (`-strict-reconfig-check`)
+
+As described in the above, the best practice of adding new members is to configure a single member at a time and verify it starts correctly before adding more new members. This step by step approach is very important because if newly added nodes have invalid peer URLs, the cluster can lose quorum. The quorum loss can happen because the new node are counted in the quorum of the cluster even the node cannot be reached from existing nodes. In addition, it can be caused by miss operation and bugs in control programs which launch etcd.
```

Comment:
Also quorum loss might happen if there is a connectivity issue or there are operational issues.